schema-bridge.graphql

```diff
@@ -100,32 +135,147 @@ type Protocol @entity {
   " The blockchain network this subgraph is indexing on "
   network: Network!
 
-  " The type of protocol (e.g. DEX, Lending, Yield, etc) "
+  " The type of protocol (e.g. DEX, Lending, Yield, Bridge, etc) "
   type: ProtocolType!
 
   ##### Quantitative Data #####
 
-  " Current TVL (Total Value Locked) of the entire protocol "
+  " Current TVL (Total Value Locked) of the entire protocol on this chain. "
   totalValueLockedUSD: BigDecimal!
 
   " Current PCV (Protocol Controlled Value). Only relevant for protocols with PCV. "
   protocolControlledValueUSD: BigDecimal
 
-  " Revenue claimed by suppliers to the protocol. LPs on DEXs (e.g. 0.25% of the swap fee in Sushiswap). Depositors on Lending Protocols. NFT sellers on OpenSea. "
+  " Revenue claimed by suppliers to the protocol. LPs on DEXs (e.g. 0.25% of the swap fee in Sushiswap). Depositors on Lending Protocols. NFT sellers on OpenSea. In the case of bridges, revenue only includes that of transactions initiated in this chain ""
   cumulativeSupplySideRevenueUSD: BigDecimal!
 
-  " Gross revenue for the protocol (revenue claimed by protocol). Examples: AMM protocol fee (Sushi’s 0.05%). OpenSea 10% sell fee. "
+  " Gross revenue for the protocol (revenue claimed by protocol). Examples: AMM protocol fee (Sushi’s 0.05%). OpenSea 10% sell fee. In the case of bridges, revenue only includes that of transactions initiated in this chain ""
   cumulativeProtocolSideRevenueUSD: BigDecimal!
 
-  " All revenue generated by the protocol. e.g. 0.30% of swap fee in Sushiswap, all yield generated by Yearn. "
+  " All revenue generated by the protocol. e.g. 0.30% of swap fee in Sushiswap, all yield generated by Yearn. In the case of bridges, revenue only includes that of transactions initiated in this chain "
   cumulativeTotalRevenueUSD: BigDecimal!
 
-  " Number of cumulative unique users "
+  " Number of cumulative unique addresses that bridged funds or messages in and out of this chain "
   cumulativeUniqueUsers: Int!
 
-  " Total number of pools "
+  ##### Snapshots #####
+
+  " Daily usage metrics for this protocol "
+  dailyUsageMetrics: [UsageMetricsDailySnapshot!]!
+    @derivedFrom(field: "protocol")
+
+  " Hourly usage metrics for this protocol "
+  hourlyUsageMetrics: [UsageMetricsHourlySnapshot!]!
+    @derivedFrom(field: "protocol")
+
+  " Daily financial metrics for this protocol "
+  financialMetrics: [FinancialsDailySnapshot!]! @derivedFrom(field: "protocol")
+
+  ##### Pools #####
+
+  " All pools that belong to this protocol "
+  pools: [Pool!]! @derivedFrom(field: "protocol")
+}
+
+type BridgeProtocol implements Protocol @entity {
+  " Smart contract address of the protocol's main contract (Factory, Registry, etc) "
+  id: ID!
+
+  " Name of the protocol, including version. e.g. Uniswap v3 "
+  name: String!
+
+  " Slug of protocol, including version. e.g. uniswap-v3 "
+  slug: String!
```

Comment:
there should be a field that will be the same in all chains for a specific bridge, this seems like what it would be. Maybe just add that to the comment

Reply:
This makes sense. It should be this same one right? Since it’s how we do it on all other protocols. I’ll add it to the comment

Reply:
yeah I would think so, just wanted to throw that out since we will probably want on ingestion side